canary/generator/download/download_whole.py

```diff
@@ -0,0 +1,38 @@
+from collections import defaultdict
+import json
+from json import JSONDecodeError
+import sys
+import os
+
+import requests
+
+from canary.generator.download.query import build_query_string
+
+if __name__ == '__main__':
+    dates_versions = requests.get(
+        'https://aggregates.telemetry.mozilla.org/aggregates_by/build_id/channels/nightly/dates/').text
+    dates_versions = json.loads(dates_versions)
```

Comment:
Small nit: Instead of re-using the variable with different content, it is probably easier to follow if you use different names for different operations, e.g.:
```python
reponse = request.get(...)
dates_versions = json.loads(response.text)
```